targets/TARGET_NXP/TARGET_LPC176X/device/TOOLCHAIN_IAR/LPC17xx.icf

```diff
@@ -12,9 +12,9 @@ define symbol __ICFEDIT_region_RAM_start__   = 0x100000C8;
 define symbol __ICFEDIT_region_RAM_end__     = 0x10007FDF;
 
 /*-Sizes-*/
-/*Heap 1/4 of ram and stack 1/8*/
-define symbol __ICFEDIT_size_cstack__   = 0x1000;
-define symbol __ICFEDIT_size_heap__     = 0x2000;
+/*Heap 1/4 of ram and stack 1/4*/
```

Comment:
is this comment true now with these update? Do we really need almost 2x bigger ISR stack (that is cstack as I recall) ?